src/Bicep.Core/Syntax/TestDeclarationSyntax.cs

```diff
@@ -55,5 +58,8 @@ this.Value switch
             this.TryGetBody() ?? throw new InvalidOperationException($"A valid test body is not available on this test due to errors. Use {nameof(TryGetBody)}() instead.");
 
         public ArtifactType GetArtifactType() => ArtifactType.Module;
+
+        public ResultWithDiagnostic<string> ResolveArtifactPath(RootConfiguration _)
+            => SyntaxHelper.TryGetForeignTemplatePath(this, x => x.PathHasNotBeenSpecified());
```

Comment:
Could you update the wording of the message? "...template file path..." feels misleading.